keyboards/aki27/cocot46plus/cocot46plus.c

```diff
@@ -0,0 +1,289 @@
+/*
+Copyright 2022 aki27
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
```

Comment:
```suggestion
#include "cocot46plus.h"
```

Reply:
After this proposed change was made, the keyboard no long compiles:

```
Compiling: keyboards/aki27/cocot46plus/cocot46plus.c                                               keyboards/aki27/cocot46plus/cocot46plus.c: In function 'process_record_kb':
keyboards/aki27/cocot46plus/cocot46plus.c:135:20: error: 'CPI_SW' undeclared (first use in this function)
  135 |     if (keycode == CPI_SW && record->event.pressed) {
      |                    ^~~~~~
keyboards/aki27/cocot46plus/cocot46plus.c:135:20: note: each undeclared identifier is reported only once for each function it appears in
keyboards/aki27/cocot46plus/cocot46plus.c:141:20: error: 'SCRL_SW' undeclared (first use in this function)
  141 |     if (keycode == SCRL_SW && record->event.pressed) {
      |                    ^~~~~~~
keyboards/aki27/cocot46plus/cocot46plus.c:146:20: error: 'ROT_R15' undeclared (first use in this function)
  146 |     if (keycode == ROT_R15 && record->event.pressed) {
      |                    ^~~~~~~
keyboards/aki27/cocot46plus/cocot46plus.c:151:20: error: 'ROT_L15' undeclared (first use in this function)
  151 |     if (keycode == ROT_L15 && record->event.pressed) {
      |                    ^~~~~~~
keyboards/aki27/cocot46plus/cocot46plus.c:156:20: error: 'SCRL_IN' undeclared (first use in this function)
  156 |     if (keycode == SCRL_IN && record->event.pressed) {
      |                    ^~~~~~~
keyboards/aki27/cocot46plus/cocot46plus.c:161:20: error: 'SCRL_TO' undeclared (first use in this function)
  161 |     if (keycode == SCRL_TO && record->event.pressed) {
      |                    ^~~~~~~
keyboards/aki27/cocot46plus/cocot46plus.c:167:20: error: 'SCRL_MO' undeclared (first use in this function)
  167 |     if (keycode == SCRL_MO) {
```

Reply:
That's because these keycodes are generated by the CLI from the list in `keyboard.json`. I guess th4 best solution here is to replace the `quantum.h` include in this header with `QMK_KEYBOARD_H`.

Reply:
Then there is nothing to do, as the change to replace "QMK_KEYBOARD_H" was not added to this branch. Marking as resolved.